internal/commands/build/build.go

```diff
@@ -145,13 +151,23 @@ func buildImageUsingBuildx(app *types.App, contextPath string, opt buildOptions,
 			Driver: d,
 		},
 	}
-	pw := progress.NewPrinter(ctx, os.Stderr, opt.progress)
+
+	var out *os.File
+	if opt.quiet {
+		if out, err = os.Create(os.DevNull); err != nil {
+			return nil, err
+		}
+	} else {
+		out = os.NewFile(dockerCli.Out().FD(), "/dev/stdout")
```

Comment:
Does `/dev/stdout` work on Windows?

Reply:
this is only a name, the important system reference is the FileDescriptor